.github/release.yml

```diff
@@ -0,0 +1,13 @@
+name: Release
```

Comment:
Oh, we made a mistake here! The file should be at:

```
.github/workflows/release.yml
```

...not...

```
.github/release.yml
```

...GitHub doesn't recognise the workflow right now because this file is in the wrong place - so 'Release' isn't listed [here](https://github.com/guardian/targeting-client/actions):

![image](https://github.com/guardian/targeting-client/assets/52038/55661a04-1288-438f-ac64-ae7bc93d1922)

Reply:
Oops, here's a PR to fix this: https://github.com/guardian/targeting-client/pull/39

Reply:
The [first release](https://github.com/guardian/targeting-client/actions/runs/7299624335/job/19892813494) failed because we changed the Maven coordinates of the artifact so now `sbt-version-policy` plugin cannot find the previous version. We've raised https://github.com/guardian/targeting-client/pull/40 to fix this.